homeassistant/components/camect/camera.py

```diff
@@ -0,0 +1,204 @@
+"""
+This platform provides support to streaming any camera supported by Camect
+Home using WebRTC.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/camera.camect/
+"""
+import asyncio
+from contextlib import suppress
+import logging
+from typing import Dict
+
+import aiohttp
+from aiohttp import web
+import async_timeout
+
+from homeassistant.components import camect, camera
+from homeassistant.components.http import HomeAssistantView
+
+REQUIREMENTS = ['camect-py==0.1.0']
```

Comment:
No reqs for platforms if components have them.

Reply:
Done.